### ISSUE TITLE
blueman (Bluetooth Manager) requires two python2 packages upon install attempt

### ISSUE THREAD

**Issue by edwardp3** [OpenMandrivaAssociation/distribution#2830]
**OpenMandriva version**:

Cooker

**Describe the bug**:

Attempted to install blueman (Bluetooth Manager). dnf is looking for packages providing python2-dbus and python2-gobject3. Intent is to pair Bluetooth headphones to a Bluetooth USB dongle.

**Logs and screenshots if relevant**

```
Error: 
 Problem: conflicting requests
  - nothing provides python2-dbus needed by blueman-1.23-4.x86_64
  - nothing provides python2-gobject3 needed by blueman-1.23-4.x86_64
(try to add '--skip-broken' to skip uninstallable packages)
```

**Comment by berolinux** [comment]
blueman is obsolete and abandoned upstream.
Try using bluedevil instead.

**Comment by edwardp3** [comment]
Thank you.I should have mentioned I have Cooker installed in a VIrtualBox.

bluedevil was already installed, was successful pairing the headphones in the VBox, but the audio continues to come out of the PC speakers.

I found _some_ posts online claiming that VirtualBox passes the audio to the host, but with the same headphones paired to the host, the audio still comes out of the speakers. I also tried with a pair of USB headphones, with the same results.
